offchain-modules/packages/x/src/handlers/eth.ts

```diff
@@ -130,7 +131,7 @@ export class EthHandler {
       return {
         id: lockRecord.txHash,
         chain: ChainType.ETH,
-        amount: lockRecord.amount,
+        amount: new Amount(lockRecord.amount, 0).sub(new Amount(lockRecord.bridgeFee, 0)).toString(0),
```

Comment:
I think add a new fee field instead of modify original data maybe  clearer and better

Reply:
how about comprehend `amount` field in this way, it means the amount to be minted